src/NumbersFromStream/Main.java

```diff
@@ -0,0 +1,18 @@
+package NumbersFromStream;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Main {
+    public static void main(String[] args) {
+        List<Integer> numbers = Stream.iterate(8, number -> number + 1)
+                .filter(number -> number > 100)
+                .filter(number -> number % 5 == 0)
+                .map(number -> number * 3)
+                .limit(10)
+                .collect(Collectors.toList());
```

Comment:
Ewentualnie `forEach` z `System.out.println`

Reply:
Próba zmiany na `forEach `z `System.out.println` kończyła się tym, że dochodziło do niezgodności typów List<Integer> kontra void.

Reply:
To nie przypisuj do zmiennej typu `List`, mówiąc żebyś wywołał `forEach` od razu miałem na myśli że możesz od razu wyświetlać te numery bez konieczności przypisywania wyniku do Listy